### PR TITLE
Initializes the Community Ads RFP

### PR DESCRIPTION
**Pull request by MxRaCohen**
Uses some examples and highlights that we're looking to branch out.

### CODE REVIEW

File: content/posts/2022-community-ads-rfp.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## Know of a worthy cause?
+
+Do these examples remind you of your own cause? Spread the word by submitting a community ad application today!
+
+* [Launch the application!](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp)
+
+
+
+Thanks again for being along with us on this journey to build an ethical ad network.
+Please [email us](mailto:ads@ethicalads.io) if you have any ideas or feedback on our product or roadmap,
```
Comment (ericholscher) [inline content/posts/2022-community-ads-rfp.md:57]: Roadmap doesn't make sense here. Seems like this sign off was copied from the newsletter?

File: content/posts/2022-community-ads-rfp.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+As you can see, these are often ads for regional technical conferences but they don't have to be!
+We'd love to run more ads supporting the developer community either by raising awareness for dev-focused charities or non-profit organizations.
+Just reach out!
+
+
+## Know of a worthy cause?
+
+Do these examples remind you of your own cause? Spread the word by submitting a community ad application today!
+
+* [Launch the application!](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp)
```
Comment (ericholscher) [inline content/posts/2022-community-ads-rfp.md:52]: This is an odd CTA, and a single bullet point feels odd. There's a better way to do this.

File: content/posts/2022-community-ads-rfp.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+Title: Community Ads Program - Request for Proposals
+Date: May 10, 2022
+description: Requests new proposals for Open Source projects to be featured across EthicalAds
+tags: community, ads
+authors: Ra Cohen
+image: /images/posts/2022-community-ads-rfp.jpg
+image_credit: <span>Photo by <a href="https://unsplash.com/@presetbase?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Presetbase Lightroom Presets</a> on <a href=" https://unsplash.com/photos/QN6NkYi3CKs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>
+
+
+Here at EthicalAds, we believe advertising can be used as a force for good, supporting Open Source and fostering community among devs across the world.
```
Comment (ericholscher) [inline content/posts/2022-community-ads-rfp.md:10]: We should decide is open source is cap'd, we don't do it consistently.

File: content/posts/2022-community-ads-rfp.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+## Seeking proposals for our Community Ads Program
+
+As part of realizing our [Advertising vision](https://www.ethicalads.io/advertising-vision/?ref=community-ads-rfp), we like to give back to the developer community by running **free** ads for open source projects and community initiatives.
+This is called our [Community Ads Program](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp).
+Historically, we've been proactive about reaching out to causes that we personally attend like PyCon, but we'd love to support other initiatives.
+Hence this blog post!
+
+Do you have an open source project that is entirely free to use that you'd like to share with the world?
+Do you run a conference or are starting a local chapter of a developer group that you'd like to advertise?
```
Comment (ericholscher) [inline content/posts/2022-community-ads-rfp.md:20]: These feel like they could be a bullet point, if we reworded them a bit.

File: content/posts/2022-community-ads-rfp.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+Let us know!
```
Comment (ericholscher) [inline content/posts/2022-community-ads-rfp.md:21]: How? This should be a link probably..

File: content/posts/2022-community-ads-rfp.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+
+## Previous Community Ads
```
Comment (ericholscher) [inline content/posts/2022-community-ads-rfp.md:24]: These ads are different sizes & aspect ratios. I'd try and keep then consistent, or at least the horizontal ones the same size.

**Review by ericholscher — COMMENTED** [review]
This is a good start, but could be cleaned up a bit. I'd also use fewer `!`'s, there's currently 6, which feels a bit much for only a few paragraphs.

File: content/posts/2022-community-ads-rfp.md (hunk 6 above)
Comment (davidfischer) [reply content/posts/2022-community-ads-rfp.md:24]: You can force them to be consistent by adding `style="max-width: 300px"` to the `img` tag.

File: content/posts/2022-community-ads-rfp.md (hunk 3 above)
Comment (davidfischer) [reply content/posts/2022-community-ads-rfp.md:10]: I do feel like this intro isn't exactly on-message. What exactly our message should be isn't precisely defined but generally:

* Community ads are for where paid ads don't make sense.
* Community ads MUST benefit the developer community.
* This means conferences and projects mostly.

File: content/posts/2022-community-ads-rfp.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+image_credit: <span>Photo by <a href="https://unsplash.com/@presetbase?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Presetbase Lightroom Presets</a> on <a href=" https://unsplash.com/photos/QN6NkYi3CKs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>
+
+
+Here at EthicalAds, we believe advertising can be used as a force for good, supporting Open Source and fostering community among devs across the world.
+
+## Seeking proposals for our Community Ads Program
+
+As part of realizing our [Advertising vision](https://www.ethicalads.io/advertising-vision/?ref=community-ads-rfp), we like to give back to the developer community by running **free** ads for open source projects and community initiatives.
+This is called our [Community Ads Program](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp).
+Historically, we've been proactive about reaching out to causes that we personally attend like PyCon, but we'd love to support other initiatives.
```
Comment (davidfischer) [inline content/posts/2022-community-ads-rfp.md:16]: Actually PyCon mostly doesn't need advertising. Instead, we did regional Python conferences where the word is harder to get out.

**Comment by ericholscher** [comment]
@MxRaCohen Any more progress here?

File: content/posts/2022-community-ads-rfp.md (hunk 2 above)
Comment (MxRaCohen) [reply content/posts/2022-community-ads-rfp.md:53]: reworked into the copy

File: content/posts/2022-community-ads-rfp.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+authors: Ra Cohen
+image: /images/posts/2022-community-ads-rfp.jpg
+image_credit: <span>Photo by <a href="https://unsplash.com/@presetbase?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Presetbase Lightroom Presets</a> on <a href=" https://unsplash.com/photos/QN6NkYi3CKs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>
+
+
+Here at EthicalAds, we believe advertising can be used as a force for good, supporting open source and fostering community among devs across the world.
+
+## Seeking proposals for our Community Ads Program
+
+As part of realizing our [Advertising vision](https://www.ethicalads.io/advertising-vision/?ref=community-ads-rfp), we like to give back to the developer community by running **free** ads for open source projects and community initiatives.
```
Comment (davidfischer) [inline content/posts/2022-community-ads-rfp.md:14]:
```suggestion
As part of realizing our [Advertising vision](https://www.ethicalads.io/advertising-vision/?ref=community-ads-rfp), we like to give back to the developer community by running **free** ads for open source projects and community initiatives like open source conferences.
```

File: content/posts/2022-community-ads-rfp.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+</div>
+
+<div class="postimage text-center">
+  <img src="{static}../images/posts/2022-pycamp.png" alt="¡Programa tus ideas sin límites durante todo un fin de semana all-inclusive!" style="max-width: 300px">
+  <p>An ad for a Spanish Python conference, PyCamp 2022.</p>
+</div>
+
+<div class="postimage text-center">
+  <img src="{static}../images/posts/2019-PyBay.png" alt="Support your SF Bay Area PyCon. Join 750+ Pythonistas in San Francisco Aug 15-18" style="max-width: 300px">
+  <p>An ad for a San Fran Python conference from 2019.</p>
```
Comment (davidfischer) [inline content/posts/2022-community-ads-rfp.md:41]:
```suggestion
  <p>An ad for a Bay Area Python conference from 2019.</p>
```

No self-respecting Californian calls it San Fran =)

File: content/posts/2022-community-ads-rfp.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+</div>
+
+As you can see, these are often ads for regional technical conferences but they don't have to be.
+We'd love to run more ads supporting the developer community either by raising awareness for dev-focused charities or non-profit organizations.
+
+
+## Know of a worthy cause?
+
+Spread the word by submitting a [community ad application](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp)today!
```
Comment (davidfischer) [inline content/posts/2022-community-ads-rfp.md:50]:
```suggestion
Spread the word by submitting a [community ad application](https://www.ethicalads.io/community-ads/?ref=community-ads-rfp) today!
```

File: content/posts/2022-community-ads-rfp.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
+Title: Community Ads Program - Request for Proposals
+Date: May 13, 2022
```
Comment (davidfischer) [inline content/posts/2022-community-ads-rfp.md:2]:
```suggestion
Date: May 16, 2022
```

**Review by davidfischer — APPROVED** [review]
I think this is just about ready with a few tweaks. Let's ship it Monday!

**Pull request merged**